.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
I think that daily interval is very bother for us.
weekly interval is better.
In other words, we wouldn't release this deamonset in weekly or daily, right?

Reply:
I don't think so, actual vulnerability reports won't be so often.
And when a vulnerability is found, we should recognize it ASAP.

Reply:
Hmm, it's reasonable. Let's keep as-is.

Reply:
Yes, "daily" is intended as above.